lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -272,11 +276,11 @@ data PoolDbData = PoolDbData
 combineDbAndLsqData
     :: forall m. (Monad m)
     => TimeInterpreter m
-    -> ProtocolParameters
+    -> Int -- ^ nOpt; desired number of pools
```

Comment:
Do we need a `newtype DesiredNumberOfPools`?

Reply:
hmm, I do not see this newtype: 
```
pawel@pure:~/Work/IOHK/cardano-wallet (master)$ ag DesiredNumberOfPools
lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs
168:    protocolParametersDesiredNumberOfPools  Word16          sql=desired_pool_number

lib/core/src/Cardano/Wallet/DB/Sqlite.hs
450:        addColumn conn True (DBField ProtocolParametersDesiredNumberOfPools) value
```